### ISSUE TITLE
7.99.12-pre3 - Model Missing errors

### ISSUE THREAD

**Issue by Dockter** [BuildCraft/BuildCraft#3850]
Hello,

Here's some model load errors you may not have seen on your end yet:

Using today's latest commit.

https://gist.github.com/Dockter/79f0ed462f6561ece90f264807ad93b6

**Comment by AlexIIL** [comment]
Yeah forge changed something somehere and suddenly there's a lot of log spam. I'm probably not going to look into it yet unless it's a problem in-game (sorry about your log file size though).

**Comment by AlexIIL** [comment]
Thanks for reporting though, I'll need to look into this at some point in the future :)

**Comment by Dockter** [comment]
I have found at least one item in the main "buildcraft" creative tab that had no name and had no model/image.  I'll try it dig into which one specifically was missing tonight.

**Comment by Dockter** [comment]
The block missing its model and texture:

buildcraftbuilders:replacer

This was the only block or item I could find that was missing a model or a texture.

**Comment by afdw** [comment]
https://github.com/BuildCraft/BuildCraft/commit/2cd4b17c1eb1193e712a867544e5f1f9115c5031
https://github.com/BuildCraft/BuildCraft/commit/f115373e81af5d1f636448b02430f8d8b03bbf03
